### ISSUE TITLE
Does it possible to use a custom domain?

### ISSUE THREAD

**Issue by numbqq** [awslabs/aws-js-s3-explorer#29]
For the normal Static Website I can use a custom domain like `www.mysite.com`, but for `aws-js-s3-explorer` I can only use `https://s3.amazonaws.com/my-BUCKET/index.html` to access the  bucket. Does it possible to use custom domain like  `www.mysite.com` to access the bucket? 

Thanks.

**Comment by dap1995** [comment]
I have the same doubt, is it possible?

**Comment by luckyshot** [comment]
> is it possible?

Yes! I'm currently running **v2-alpha** in my own domain and it seems to work fine. 

The only thing you need to do is to add the [CORS policy here](https://github.com/awslabs/aws-js-s3-explorer/tree/v2-alpha#enabling-cors), just remember to modify it to your needs AND add your custom domain as an `AllowedOrigin` too.

**Comment by john-aws** [comment]
Closing for lack of engagement. Please re-open if we can help.
